### PR TITLE
Use `bundle add` in README installation instructions

### PR DESCRIPTION
**Pull request by glaucocustodio**
As per https://github.com/rubygems/rubygems/pull/5337, we can simplify the steps of adding a gem.

### CODE REVIEW

**Review by deivid-rodriguez — COMMENTED** [review]
In this case, since we're are also providing instructions to install the unreleased gem from the master branch directly, I think we should either migrate both alternatives to use `bundle add`, or leave things as they are.

What do you think?

**Comment by glaucocustodio** [comment]
I can't get `bundle add ransack --git=git@github.com:activerecord-hackery/ransack.git` working, I get:

```sh
[!] There was an error parsing `injected gems`: You cannot specify the same gem twice with different version requirements.
You specified: ransack (~> 2.4.2) and ransack (>= 0). Gem already added. Bundler cannot continue.

 #  from injected gems:1
 #  -------------------------------------------
 >  gem "ransack", ">= 0", :git => "git@github.com:activerecord-hackery/ransack.git"
```

**Comment by glaucocustodio** [comment]
Anyway I think is valid having `bundle add` to let people know about this :)

**Comment by deivid-rodriguez** [comment]
Mmmm, maybe a `bundle add` issue? If we don't migrate both instructions, I think it makes things more confusing that they currently are to be honest.

**Comment by glaucocustodio** [comment]
Why do we recommend people using the master version? This is odd, as any breaking change would affect people pointing to it..

File: README.md [hunk 1]
```diff
@@ -17,17 +17,16 @@ or controller layer, you're probably not looking for Ransack.
 
 Ransack is supported for Rails 7.0, 6.x on Ruby 2.6.6 and later.
 
-In your Gemfile, for the last officially released gem:
+Run to install:
```
Comment (deivid-rodriguez) [inline README.md:20]: Can we say that the command below also adds the gem to the `Gemfile` automatically? Maybe something like

```suggestion
To install `ransack` and add it to your Gemfile, run
```

**Review by deivid-rodriguez — COMMENTED** [review]
Minor wording issue, but looks good to me now.

**Pull request merged**